core/domain/draft_upgrade_services.py

```diff
@@ -312,6 +312,123 @@ def _convert_html_in_draft_change_list(
+                        exp_change
+                    )
+                    answer_group_dicts = (
+                        edit_interaction_answer_groups_cmd.new_value
+                    )
+                    for answer_group in answer_group_dicts:
+                        if (
+                            answer_group['tagged_skill_misconception_id']
+                            is not None
+                        ):
+                            answer_group['tagged_skill_misconception_id'] = None
+                        if len(answer_group['rule_specs']) == 0:
+                            answer_group_dicts.remove(answer_group)
+                        if (
+                            answer_group['outcome']['labelled_as_correct'] and
+                            answer_group['outcome'][
+                                'dest'] == exp_change.state_name
+                        ):
+                            answer_group['outcome'][
+                                'labelled_as_correct'] = False
+                        if (
+                            answer_group['outcome']['refresher_exploration_id']
+                            is not None
+                        ):
+                            answer_group['outcome'][
+                                'refresher_exploration_id'] = None
+                elif exp_change.property_name == (
+                    exp_domain.STATE_PROPERTY_CONTENT
+                ):
+                    # Here we use cast because this 'if' condition forces
+                    # change to have type EditExpStatePropertyContentCmd.
+                    edit_contents_cmd = cast(
+                        exp_domain.EditExpStatePropertyContentCmd,
+                        exp_change
+                    )
+                    html = edit_contents_cmd.new_value['html']
+                    html = exp_domain.Exploration.fix_rte_tags(html)
+                    html = exp_domain.Exploration.fix_tabs_and_collapsible_tags(
+                        html)
+                    edit_contents_cmd.new_value['html'] = html
+                elif exp_change.property_name == (
+                    exp_domain.STATE_PROPERTY_WRITTEN_TRANSLATIONS
+                ):
+                    # Here we use cast because this 'if' condition forces
+                    # change to have type EditExpStatePropertyWritten
+                    # TranslationsCmd.
+                    edit_interaction_written_translations_cmd = cast(
+                        exp_domain.EditExpStatePropertyWrittenTranslationsCmd,
+                        exp_change
+                    )
+                    written_translations = (
+                        edit_interaction_written_translations_cmd.new_value)
+                    for translations in (
+                        written_translations['translations_mapping'].values()
+                    ):
+                        for written_translation in translations.values():
```

Comment:
Would this be the same as:
```
for written_translation in written_translations['translations_mapping'].values().values():
```
Not sure... but might make the code more concise

Reply:
It didn't pass, was not able to use 2 `values()` together.
Thankd